app/src/main/java/com/github/sdp_begreen/begreen/activities/SignInActivity.kt

```diff
@@ -35,13 +40,14 @@ class SignInActivity : AppCompatActivity() {
         if (result.resultCode == Activity.RESULT_OK) {
```

Comment:
When I tried to sign in, it didn't send me to the main activity as expected. After debugging a while I noticed that the problem comes from this condition. For some reason, the _result.resultCode_ is **CANCELLED** even if I (as a user) did everything right.

Below you can see the state of the debugger.

Try to fix this problem or tell me if I am the only one who has it

<img width="1124" alt="image" src="https://user-images.githubusercontent.com/91261122/228348938-d5e45565-f5bc-436d-993b-754762cb434d.png">

Reply:
See my comments below about SHA 1. It should resolve the problem. Let me know if you have any questions.